### DISCUSSION TITLE
Explore decentralized/web3 use cases for HC and beyond (22Q2 brainstorming)

### DISCUSSION THREAD

**Discussion by jsoares** [consensus-shipyard/consensuslab#144] [Project proposals]
- Decentralized Github
- #75 
  - Barath at Network Research took over the Ivy work; I think the conclusion was to move away from it

@adlrocha @quema @dnkolegov @guy-goren

**Comment by adlrocha** [comment]
> Barath at Network Research took over the Ivy work; I think the conclusion was to move away from it

Is there any public doc where I can read the outcomes? Thanks!

**Reply by jsoares** [reply]
The project pivoted into a broader discussion around improving internet architecture, the first step being https://github.com/protocol/research-grants/blob/master/RFPs/rfp-011-changing-the-internet.md

I'll try to scavenge more information. This isn't to say the initial idea cannot be pursued by other teams (inc. ours). But we should understand the thought that went into it on the NR side. Personally, and from a networking perspective, I think the overhead/complexity of the initial vision was just untenable, with no obvious path towards a workable solution. But there may be an out-of-the-box direction waiting in the shadows.
